src/main/java/de/fearnixx/jeak/service/controller/testImpls/TestController.java

```diff
@@ -14,7 +14,7 @@ public DummyObject hello() {
         return new DummyObject("Finn", 20);
     }
 
-    @RequestMapping(method =  RequestMethod.GET, endpoint = "/info/:name")
+    @RequestMapping(method =  RequestMethod.GET, endpoint = "/info")
```

Comment:
Wasn't this previously an example of a path parameter instead of a request parameter?